src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -111,6 +112,25 @@ export const fetchDictionaryConcepts = data => (dispatch) => {
         dispatch(isFetching(false));
       });
 };
+
+export const releaseHead = (url, data) => (dispatch) => {
+  return api.dictionaries
+    .realisingHeadVersion(url, data)
+    .then(
+      (payload) => {
+        dispatch(realisingHeadSuccess(payload));
+        dispatch(isFetching(false));
+        notify.show(
+          'Head Version Successfully Released',
+          'success', 6000,
+        );
+      }
+    )
+    .catch(() => {
+      dispatch(isFetching(false));
+      notify.show("Network Error. Please try again later!", 'error', 6000);
```

Comment:
@kmunene, this seems to be a put request which seems like one that requires the authorization of some sort. Is this the only error possibility, `network error`?

Reply:
If one is either not logged in or not the owner of that dictionary they are not able to view the "Release Head Version" so they cannot make the put request.